ui/components/multichain/address-copy-button/index.scss

```
@@ -1,4 +1,7 @@
 .multichain-address-copy-button {
+  min-height: 32px;
+  height: auto;
+
   &__address--wrap {
     word-break: break-word;
```

Should the height auto styles be tied to the wrap prop?
```suggestion

  &__address--wrap {
    word-break: break-word;
    min-height: 32px;
    height: auto;
```